Lib/wave.py

```diff
@@ -429,15 +429,15 @@ class Wave_write:
 
```

Comment:
I think that the simplest way is to add `_file` as a class variable.
```suggestion

    _file = None

```